app/models/notice_builder.rb

```diff
@@ -33,6 +34,31 @@ def set_json_params
     ).deep_symbolize_keys
   end
 
+  def set_works_json
```

Comment:
Can we add a spec that will make sure that the `works_json` field is set properly?